pkg/storaged/utils.js

```diff
@@ -624,6 +624,30 @@
     return children;
 }
 
+export function get_all_children(client, path) {
+    const children = get_children(client, path);
+
+    if (client.blocks[path]) {
+        const mdraid = client.blocks[path].MDRaidMember;
+        if (mdraid != "/")
+            children.push(mdraid);
+    }
+
+    if (client.blocks_pvol[path]) {
+        const vgroup = client.blocks_pvol[path].VolumeGroup;
+        if (vgroup != "/")
+            children.push(vgroup);
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18698-20230424-134533-fc33405c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)